components/raftstore/src/store/peer_storage.rs

```diff
@@ -835,10 +847,22 @@ where
     }
 
     #[inline]
-    pub fn committed_index(&self) -> u64 {
+    pub fn commit_index(&self) -> u64 {
         self.raft_state.get_hard_state().get_commit()
     }
 
+    #[inline]
+    pub fn set_commit_index(&mut self, commit: u64) {
+        if commit > self.commit_index() {
```

Comment:
When will `commit < self.commit_index()`?

Reply:
For now, the answer is no so I change to `assert!(commit >= self.commit_index())`